### PR TITLE
Update laravel guide for v.0.0.2

### LINKED ISSUE #350: failed to run 'composer require open-telemetry/opentelemetry' in Integrating Opentelemetry PHP into Laravel Applications
I have succeeded run the step 1 in this guide. But every time I run the command in step 2, I will always fail to run it. I am new to php, and I have no idea to solve this problem.

image

### PR DESCRIPTION
**Pull request by prondubuisi**
Update Laravel integration guide to meet V.0.0.2 requirements.
This fixes #350

### CODE REVIEW

**Review by tydhot — APPROVED** [review]
Thanks!

File: docs/laravel-integration.md [hunk 1]
```diff
@@ -88,24 +95,26 @@ $kernel->terminate($request, $response);
  To use open-telemetry specific classes within our application we have to import them at the top of our index file, using the `use` keyword. This is what our imports look like:
```
[the remark this answers is not in the document]
Comment (bobstrecansky) [reply docs/laravel-integration.md:95]: Yes, or maybe something along the lines of:

`This is what our list of imported libraries should look like`

[the remark this answers is not in the document]
Comment (prondubuisi) [reply docs/laravel-integration.md:95]: Spot on @tydhot @bobstrecansky, I am updating this. Thanks.

**Comment by prondubuisi** [comment]
Updated @bobstrecansky @tydhot, I hope the imports are clearer now.

**Comment by prondubuisi** [comment]
Updated @bobstrecansky @tydhot, I hope the imports are clearer now.

**Comment by tydhot** [comment]
> Updated @bobstrecansky @tydhot, I hope the imports are clearer now.

thanks a lot, it is worked now:)

**Comment by prondubuisi** [comment]
Thanks for helping with the review @tydhot .

**Review by bobstrecansky — APPROVED** [review]

**Pull request merged**